### PR TITLE
docs: add TiKV adopters and Roadmap

### PR DESCRIPTION
**Pull request by queenypingcap**
## What have you changed? (mandatory)
I have added the TiKV adopters and copied the TiKV roadmap to this repo so that TiKV can have its own roadmap

## What are the type of the changes? (mandatory)

Improvement

## How has this PR been tested? (mandatory)

N/A

## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No.

## Does this PR affect tidb-ansible update? (mandatory)

No.

## Refer to a related PR or issue link (optional)

https://github.com/pingcap/docs/pull/525 

## Benchmark result if necessary (optional)

No.

## Add a few positive/negative examples (optional)

N/A

### CODE REVIEW

**Comment by ceohockey60** [comment]
Made some edits to the adopters.md table.
@queenypingcap PTAL re formatting and cleanliness.
Content LGTM.

File: docs/adopters.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+category: adopters
+---
+
+# Tikv Adopters
+
+This is a list of TiDB adopters in various industries.
+
+|Company| Industry| Success Story| 
+| --- | --- | --- |
+|[Mobike](https://en.wikipedia.org/wiki/Mobike)|Ridesharing|[English](https://www.pingcap.com/blog/Use-Case-TiDB-in-Mobike/); [Chinese](https://www.pingcap.com/cases-cn/user-case-mobike/)|
```
Comment (siddontang) [inline docs/adopters.md]: do we need to sort the company by name?

Comment (queenypingcap) [reply docs/adopters.md]: Let's just keep it as it is.  Kevin reordered them in terms of brand recognition.

**Comment by ceohockey60** [comment]
LGTM

File: docs/adopters.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+title: TiKV Adopters
+category: adopters
+---
+
+# TiKV Adopters
+
+This is a list of TiDB adopters in various industries.
```
Comment (breeswish) [inline docs/adopters.md]: The title said "TiKV Adopters" but the description becomes "TiDB adopters". May cause confusion.

File: docs/ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- [x] Transaction
+    - [x] Optimize transaction conflicts
+    - [ ] Distributed GC
+- [x] Coprocessor
+    - [x] Streaming
+- [ ] Tool
+    - [x] Import distributed data
+    - [ ] Export distributed data
+    - [ ] Disaster Recovery
+    - [ ] TiKV client (Rust crate)
```
Comment (Hoverbear) [inline docs/ROADMAP.md]: This is present twice.

**Pull request merged**